XenAdmin/TabPages/PerformancePage.cs

```diff
@@ -78,6 +78,7 @@ protected override void Dispose(bool disposing)
             if (_disposed)
                 return;
 
+            DeregisterEvents();

```

Comment:
I missed this earlier. I believe these two line should be within the `if (disposing)` block.